setup/centos5/resources/jenkins.sysconfig

```diff
@@ -3,5 +3,5 @@ JENKINS_SLAVE_JAR="/home/iojs/slave.jar"
 JENKINS_SLAVE_LOG="/home/iojs/$NAME.log"
 JENKINS_SECRET="{{secret}}"
 JENKINS_SLAVE_ARGS="-jnlpUrl https://ci.nodejs.org/computer/iojs-digitalocean-centos6-64-{{id}}/slave-agent.jnlp -secret $JENKINS_SECRET"
-JENKINS_ENV="DESTCPU={{arch}} ARCH={{arch}} OSTYPE=linux-gnu NODE_COMMON_PIPE=/home/iojs/test.pipe"
+JENKINS_ENV="PRE_1_PATH=/home/iojs/bin:$PATH JOBS=$(getconf _NPROCESSORS_ONLN) DESTCPU={{arch}} ARCH={{arch}} OSTYPE=linux-gnu NODE_COMMON_PIPE=/home/iojs/test.pipe"
```

Comment:
the path shuffling probably benefits from living in below jenkins_path (which with scl gets the devtoolset stuff) for clarity.